fedimint-dbtool/src/lib.rs

```diff
@@ -249,3 +269,10 @@ async fn open_db(options: &Options) -> fedimint_core::db::Database {
         .unwrap()
         .into_database()
 }
+
+async fn open_db_ro(options: &Options) -> fedimint_core::db::Database {
```

Comment:
Weird one-of. Shouldn't we either already have this or use it in more places right away?